doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+TODO: add highlights
+
+This release includes
+
+Backwards incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: add examples of how to deal with these changes
+
+-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Refactor regridding (:pull:`2231`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `linear_extrapolate` in recipe checks (:pull:`2324`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `unstructured_nearest` in recipe checks (:pull:`2336`) by :user:`schlunma`
```

Comment:
I think this is part of the same work as the pull request above, so i would make it a single entry

Reply:
Given the brilliant automation already setup here, would it be possible to leave this as is? To make it easier/clearer for future (and current! 😂) release managers as a precedent?

Reply:
I'm not sure if it is that brilliant, see https://github.com/ESMValGroup/ESMValCore/pull/2414#discussion_r1598614286, but it's fine to leave this as-is if you prefer.